index.bs

```diff
@@ -464,11 +513,19 @@ The <dfn method for=CookieStore>getAll(|options|)</dfn> method, when invoked, mu
 
 1. Let |origin| be |environment|’s [=/origin=].
 1. If |origin| is an [=opaque origin=], then return a new promise rejected with a "{{SecurityError}}" {{DOMException}}.
+1. Let |url| be the [=current settings object=]'s [=creation URL=].
+1. If |options|' {{CookieStoreGetOptions/url}} dictionary member is present, then run these steps:
+    1. Let |parsed| be the result of running the [=basic URL parser=] on |options|' {{CookieStoreGetOptions/url}} dictionary member with |url|.
+    1. If the [=current global object=] is a {{Window}} object and |parsed| does not [=url/equal=] |url|,
+        then return a new promise rejected with an "{{InvalidStateError}}" {{DOMException}}.
```

Comment:
I think we agreed on TypeError for all the errors, in https://github.com/WICG/cookie-store/issues/5